deploy/charts/cert-manager/README.template.md

```diff
@@ -364,17 +364,24 @@ config:
   kubernetesAPIQPS: 9000
   kubernetesAPIBurst: 9000
   numberOfConcurrentWorkers: 200
+  enableGatewayAPI: true
+  # Feature gates as of v1.16.0. Listed with their default values.
+  # See https://cert-manager.io/docs/cli/controller/
   featureGates:
-    AdditionalCertificateOutputFormats: true
-    DisallowInsecureCSRUsageDefinition: true
-    ExperimentalCertificateSigningRequestControllers: true
-    ExperimentalGatewayAPISupport: true
-    LiteralCertificateSubject: true
-    SecretsFilteredCaching: true
-    ServerSideApply: true
-    StableCertificateRequestName: true
-    UseCertificateRequestBasicConstraints: true
-    ValidateCAA: true
+    AdditionalCertificateOutputFormats=true # BETA - default=true
+    AllAlpha=false # ALPHA - default=false
+    AllBeta=false # BETA - default=false
+    ExperimentalCertificateSigningRequestControllers=false # ALPHA - default=false
+    ExperimentalGatewayAPISupport=true # BETA - default=true
+    LiteralCertificateSubject=true # BETA - default=true
+    NameConstraints=false # ALPHA - default=false
+    OtherNames=false # ALPHA - default=false
+    SecretsFilteredCaching=true # BETA - default=true
+    ServerSideApply=false # ALPHA - default=false
+    StableCertificateRequestName=true # BETA - default=true
+    UseCertificateRequestBasicConstraints=false # ALPHA - default=false
+    UseDomainQualifiedFinalizer=false # ALPHA - default=false
+    ValidateCAA=false # ALPHA - default=false
```

Comment:
This isn't valid yaml `AllAlpha=false` > `AllAlpha: false`.

I suggest removing all these featureFlag changes

Reply:
Ah oops sorry. This was from a copy paste where I was trying to define these as strings. I've fixed them to be valid YAML but can also drop them if it is better.